lib/transmuxer.js

```diff
@@ -1009,13 +1102,33 @@ Transmuxer = function() {
     self = this,
     videoTrack,
     audioTrack,
-    config,
-    pps,
+    earliestDTS,
 
     packetStream, parseStream, elementaryStream,
     aacStream, h264Stream,
     videoSegmentStream, audioSegmentStream;
 
+  this.dataEmitter = {
```

Comment:
Tiny `dataEmitter` stream to re-emit data it gets via `push` as the `Transmuxer`